main/src/com/google/refine/exporters/OdsExporter.java

```diff
@@ -76,7 +76,13 @@ public void export(final Project project, Properties params, Engine engine,
             @Override
             public void startFile(JsonNode options) {
                 table = OdfTable.newTable(odfDoc);
-                table.setTableName(ProjectManager.singleton.getProjectMetadata(project.id).getName());
+                String tableName = ProjectManager.singleton.getProjectMetadata(project.id).getName();
+                table.setTableName(tableName);
+                if(!tableName.equals("Sheet1") && odfDoc.getTableList().get(0).getTableName().equals("Sheet1")){
+                    OdfTable defTable;
+                    defTable = odfDoc.getTableByName("Sheet1");
+                    defTable.remove();
+                }
```

Comment:
Thanks, that seems to work! However, this is likely to break if the default sheet is not named `Sheet1`, which could potentially happen in a later version (for instance because the name gets localized? `Feuille1` in French?), or if the default table somehow gets moved to index 1 (because the new table gets created at index 0, maybe?).

I can think of other approaches:
* before calling `OdfTable.newTable`, delete all existing tables in the document (I suspect that fails because we must always have at least one table in the document)
* after calling `OdfTable.newTable` and setting its name, loop through all the tables in the document and delete those whose name is not the name we just set - that I think should work and should be more reliable.